### PR TITLE
Updated events

### PR DESCRIPTION
**Pull request by sbglasius**
+ bumped gradle to 2.8

### CODE REVIEW

File: site/src/site/pages/index.groovy [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import model.Event
 
 layout 'layouts/main.groovy', true,
-        pageTitle: 'The Groovy programming language',
+        pageTitle: 'The Grails Framework',
```
Comment (PascalSchumacher) [inline site/src/site/pages/index.groovy]: ???

Comment (sbglasius) [inline site/src/site/pages/index.groovy]: Whoops.. well spotted. I'll get that fixed right now :-)

**Comment by sbglasius** [comment]
@PascalSchumacher that's fixed (and my ears are still red...)

**Comment by PascalSchumacher** [comment]
Thanks!

**Pull request merged**